src/hooks/useAccountBalance.ts

```diff
@@ -102,7 +115,7 @@ export const useAccountBalance = ({
       return formatUnits(BigInt(balanceAsCoin.amount), decimals);
     }
     return undefined;
-  }, [addressOrDenom, chainId, rpc]);
+  }, [addressOrDenom, bech32AddrPrefix, decimals, dydxAddress, rpc]);
```

Comment:
[q] this dependency array looks more correct this way, but we (unfortunately) have a few cases of intentionally incomplete dependency arrays to prevent infinite renders/api calls (which again, are unfortunately not well documented). have we QA'ed this to make sure this won't happen?

Reply:
This is unintentional, im reverting, we shouldn't touch cosmos related code in this PR

Reply:
thank you!